docs/source/getting-started/running-notebooks/index.rst

```diff
@@ -0,0 +1,139 @@
+How to Run Tutorials
+====================
+
+
+Throughout the MLflow documentation, you'll find a collection of notebook-based tutorials that will help to provide a primer on learning concepts of MLflow. 
+From the very `first getting started tutorials <../intro-quickstart/index.html>`_ to in-depth use-case driven examples of using MLflow with popular ML frameworks 
+`like transformers <../../llms/transformers/index.html>`_, these tutorials 
+typically integrate with the MLflow Tracking Server (to learn more about it, see `the guide here <../../tracking.html>`_ and view the 
+`5 minute quickstart tutorial here <https://www.mlflow.org/docs/latest/getting-started/tracking-server-overview/index.html>`_) to record information about 
+experiments, runs, and the artifacts that are produced by your machine learning code. 
+
+This brief guide will walk you through some options that you have to run these tutorials and have a Tracking Server that is available to log the results to, 
+as well as providing guidance on these different modes and how to access the MLflow UI to view the results. 
+
+The listing of topics in this guide are in ascending order of complexity, so if you're looking for the quickest and easiest way to get started, please 
+evaluate the first option. 
+
+Managed MLflow
```

Comment:
Can we use `Databricks` instead of `Managed MLflow`? Feel like this section is just using different versions of Databricks, and it's kind of confusing compared to `self-managed MLflow` as they're all managed.